tmcprototype/SdpSubarrayLeafNode/SdpSubarrayLeafNode/SdpSubarrayLeafNode.py

```diff
@@ -83,8 +83,7 @@ def commandCallback(self, event):
     # Device Properties
     # -----------------
     SdpSubarrayNodeFQDN = device_property(
```

Comment:
Property name should be SdpSubarrayFQDN.

Reply:
Done.

Reply:
Commit ID: 9ab1a8bbeec93788ee716d8c14cabd8cb9e7a1bd